app/frontend/src/styles/application/jobseekers/vacancy.scss

```diff
@@ -14,12 +14,27 @@
     }
   }
 
-  .similar-jobs {
-    .govuk-grid-row {
-      align-items: stretch;
-      display: flex;
-      flex-wrap: wrap;
-      margin: 0;
+  .similar-job {
```

Comment:
changing (reverting) this css like this wont work as there will likely be content side by side of differing heights that should have a uniform height orange left border